src/ghost.js

```diff
@@ -17,9 +17,16 @@ class GhostThread extends EventEmitter {
     this._backlog = new Set() // set of past messages
     this._backlogLimit = opts.ghostBacklogLimit || DEFAULT_BACKLOG_LIMIT
 
+    this._filters = opts.ghostFilters || []
+
     this._room.on('message', async ({ from, data }) => {
       const { payload, issuer } = await this._verifyData(data)
-      if (payload) {
+
+      // looping thru _filters array w map should return an array of booleans
+      // reduce that boolean array to one boolean that indicates whether we pass all the filters
+      const passesFilters = this._filters.map(filter => filter({ payload, issuer, from })).reduce((sum, next) => sum && next, true)
```

Comment:
Can't we just make the filter function take three params instead of one? I think this would make it a little bit easier to implement filters for developers. 

Btw, you can also simplify this line by removing the `.map`.
Would look something like: 
```js
const passesFilters = this._filters.reduce((acc, filter) => acc && filter({ payload, issuer, from }), true)
```
If we change the filter params as described above:
```js
const passesFilters = this._filters.reduce((acc, filter) => acc && filter(payload, issuer, from), true)
```